src/Backend/Modules/Pages/Js/Pages.js

```diff
@@ -342,6 +342,12 @@ jsBackend.pages.extras = {
     })
 
     $('#blockHtml').unbind('show.bs.modal').on('show.bs.modal', function (e) {
+      if (jsData.Core.preferred_editor !== 'ck-editor') {
+        $('#html').val(previousContent);
```

Comment:
I don't get this. 
Why restore the previousContent?

Reply:
this loads the current html in the editor